src/index.html

```diff
@@ -240,7 +240,9 @@ <h2 class="pageItemTitle" style="display: block;">
                                     <knockout data-bind="template: { name: 'pokemonNameTemplate', data: { 'pokemon': Battle.enemyPokemon() } }">Pokémon name</knockout>
                                     <knockout data-bind="template: { name: 'caughtStatusTemplate', data: {'status': PartyController.getCaughtStatus(Battle.enemyPokemon().id)}}"></knockout>
                                     <knockout style="position: relative; top: -1px;"
-                                        data-bind="template: { name: 'pokerusStatusTemplate', data: { 'pokerus': PartyController.getPokerusStatus(Battle.enemyPokemon().id) }}">
+                                        data-bind="template: { name: 'pokerusStatusTemplate', data: {
+                                            'pokerus': PartyController.getPokerusStatus(DungeonBattle.enemyPokemon().id),
+                                            'evs': App.game.party.getPokemon(DungeonBattle.enemyPokemon().id)?.evs() }}">
```

Comment:
You should be using `Battle.enemyPokemon().id` here and not `DungeonBattle.enemyPokemon().id`

Reply:
thank you